composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package com.majotyler.hiittimer.presentation
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+class TimerViewModel : ViewModel() {
+    private val _state = MutableStateFlow(value = TimerViewState(seconds = 0))
+    val state = _state.asStateFlow()
```

Comment:
This is a little weird - but important.

`_state` can be changed. That is why it is `private`. We don't want `TimerScreen` to be able to change `_state`, and it can't this way.

`state` cannot be changed. It can only be observed. That is why it is `public` (in Kotlin if you don't specify visibility it is just public).

`state` is just a copy of `_state` that cannot be changed.